Vagrantfile

```diff
@@ -5,13 +5,82 @@
 # configures the configuration version (we support older styles for
 # backwards compatibility). Please don't change it unless you know what
 # you're doing.
+
+
+def os_is_osx?
+  `uname`.strip == 'Darwin'
```

Comment:
No more shelling out from the Vagrantfile: https://github.com/SUSE/scf/blob/harts-vagrant-use-public-network-rebased/Vagrantfile